simplekv/__init__.py

```diff
@@ -259,6 +262,20 @@ def _open(self, key):
         """
         raise NotImplementedError
 
+    def _copy(self, source, dest):
+        """Implementation for :meth:`~simplekv.CopyMoveMixin.copy`. The default
+        implementation will simply get the data from source and put it under the new key.
+        """
+        return self.put(dest, self.get(source))
+
+    def _move(self, source, dest):
+        """Implementation for :meth:`~simplekv.CopyMoveMixin.move`. The default
+        implementation will copy the data from source to dest and then delete the source key.
+        """
+        k = self._copy(source, dest)
+        self.delete(source)
+        return k
+
```

Comment:
From my understanding, `_copy()` and `_move()` should only be defined in the backends, but not in the base class. The fallback implementation should be in the `CopyMoveMixin`.